ledger-service/http-json/src/main/resources/logback.xml

```diff
@@ -9,7 +9,7 @@
                 <!-- encoders are assigned the type
                     ch.qos.logback.classic.encoder.PatternLayoutEncoder by default -->
                 <encoder>
-                    <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg %replace(, context: %marker){', context: $', ''} %n</pattern>
+                    <pattern>%date{"dd-MM-yyyy HH:mm:ss.SSS", UTC} [%thread] %-5level %logger{36} - %msg %replace(, context: %marker){', context: $', ''} %n</pattern>
```

Comment:
Why UTC? Is this synced up with other default `logback.xml` files?

Reply:
I think yes, however I didn't directly check (I just know that other services log in UTC too).

Reply:
this is what pkvutils uses afaict. Open to syncing it up with something else as well. This just seemed like the most sensible option given that it’s what most of our customers use

Reply:
Perfect, thanks!